app/javascript/multi-stamp-tubes/components/MultiStampTubes.vue

```diff
@@ -204,13 +212,14 @@ export default {
     //   ...
     // }
     targetWells() {
-      const wells = {}
-      for (let i = 0; i < this.validTransfers.length; i++) {
-        wells[this.validTransfers[i].targetWell] = {
-          pool_index: this.validTransfers[i].tubeObj.index + 1,
+      return this.validTransfers.reduce((acc, transfer) => {
+        const tubeIndex = transfer.tubeObj.index
+        acc[transfer.targetWell] = {
+          colour_index: this.colourIndex(tubeIndex),
+          human_barcode: this.tubes[tubeIndex].labware.labware_barcode.human_barcode,
```

Comment:
```suggestion
          human_barcode: transfer.tubeObj.tube.labware_barcode.human_barcode
```

Looks to me like the above should work, and is neater to get it from the variable `transfer` I think than having to use `this.tubes`?